unlock-app/src/stories/creator/CreatorLockForm.stories.js

```diff
@@ -22,17 +22,13 @@ storiesOf('CreatorLockForm', module)
     // TODO: implement this
     const lock = {
       keyPrice: '10000000000000000000',
-      expirationDuration: '172800',
+      expirationDuration: 172800,
```

Comment:
Not sure why this change is part of the PR? Could you please clarify?

Reply:
it generated a console warning, which I assumed was due to code changes in this PR since I hadn't seen it before

Reply:
Let's remove this from this PR then.

Reply:
ok

Reply:
this is needed because we are using `{...lock}` whereas prior, the lock was not being passed in at all, and props require a number. It was an error that was present but never triggered, so we do need it after all

Reply:
ok that works.